Mozu/core-theme
# Anonymous Order Status Checking

### Feature: Anonymous Order Status Checking

We’ve added a new feature that allows customers to look-up and access their order information from a link on the storefront or in their Order Confirmation email.   This option is available to customers even if they have not logged into My Account, or have placed their order anonymously through guest checkout.  

Customers are asked to provide their Order number and an additional identifying piece of information such as zip code, phone number or email address to access the order.
#### Additions
- We added a new builtin page type, `my-anonymous-account`, a view of My Account that only allows order status viewing
- We added a Hypr template for this page type, `templates/pages/my-anonymous-account`
- We modified `templates/pages/login` to offer an option to login to this anonymous mode with your order status and identifying info
- We modified relevant email templates and navigation items to include links to this new functionality.
#### Integration Options

**The preferred upgrade path is always to update your theme to extend Core8.**

If doing so results in unrelated or unpredictable regressions, then instead you can inspect the changes we made to the Core Theme and manually integrate them.
#### Upgrade Process
- Extend Core8 or merge the files as described.
- Recompile your theme using the build tools.
- Perform regression tests.
- Test the functionality by:
  - Place an order as an anonymous user. You should see a link to Anonymous Order Status in the email autoresponder you receive. Click it and confirm that you can view anonymous orders tatus.
  - Go to the `/user/login` page. Note that there is an anonymous order status login option available.
